src/components/calcite-pick-list/calcite-pick-list.e2e.ts

```diff
@@ -24,7 +24,7 @@ describe("calcite-pick-list", () => {
         const pickList = await page.find("calcite-pick-list");
         const item1 = await pickList.find("[value=one]");
         const item2 = await pickList.find("[value=two]");
-        const toggleSpy = await pickList.spyOnEvent("calcitePickListSelectionChange");
+        const toggleSpy = await pickList.spyOnEvent("calciteListSelectionChange");
```

Comment:
Could this be simplified to `calciteListChange` or is that too generic? 

Sidebar, there's no `<calcite-list>` component in the works, right? Just thinking whether there'd be a potential event name clash.

Reply:
> Sidebar, there's no <calcite-list> component in the works, right?

Not that I know of. Could there be in `calcite-components` at some point?